### ISSUE TITLE
WebDriver (WebDriverIO 5) scrollTo does not work due to deprecated API

### ISSUE THREAD

**Issue by LukoyanovE** [Codeception/CodeceptJS#1517]
#### What are you trying to achieve?
Run test with `scrollTo` on safari 12 and firefox 65 with WebDriver helper (WebDriverIO 5)

#### What do you get instead?

> Provide console output if related. Use `--verbose` mode for more details.

```bash
TypeError: this.browser.getElementLocation is not a function
      at WebDriver.scrollTo (node_modules/codeceptjs/lib/helper/WebDriver.js:1248:43)
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:229:7)
```

### Details

* CodeceptJS version: 2.0.4
* NodeJS Version: 8.14.3
* Operating System: MacOS High Sierra
* WebDriver

It's all about WebDriverIO 5 breaking changes:
`const location = await this.browser.getElementLocation(elementId);`
`getElementLocation` was deprecated. `element.getLocation` should be used now.
https://webdriver.io/docs/api/element/getLocation.html

I have a variant to fix it. But will do it in a few days, if someone will not do it earlier.

**Comment by PeterNgTr** [comment]
I think it is the same
```
TypeError: this.browser.getLocation is not a function
      at WebDriver.scrollTo (node_modules/codeceptjs/lib/helper/WebDriver.js:1248:43)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by PeterNgTr** [comment]
@LukoyanovE  If you want, I can create a PR for this?

**Comment by LukoyanovE** [comment]
@PeterNgTr , 
> if someone will not do it earlier.

If you want or if you need this as soon as possible, do.

I can start to fix it on monday, unless anybody fix.

**Comment by PeterNgTr** [comment]
Not huge changes, I could make it. https://github.com/Codeception/CodeceptJS/pull/1519

**Comment by LukoyanovE** [comment]
Fixed in #1519
